firstvoices/backend/tests/test_apis/test_widgets_api.py

```diff
@@ -5,10 +5,12 @@
 from backend.models.constants import AppRole, Visibility
 from backend.models.widget import SiteWidget, WidgetSettings
 from backend.tests import factories
-from backend.tests.test_apis.base_api_test import BaseControlledSiteContentApiTest
+from backend.tests.test_apis.base_api_test import (
+    BaseReadOnlyControlledSiteContentApiTest,
+)
 
 
-class TestSiteWidgetEndpoint(BaseControlledSiteContentApiTest):
+class TestSiteWidgetEndpoint(BaseReadOnlyControlledSiteContentApiTest):
```

Comment:
Same comment about `BaseControlledSiteContentApiTest` here.

Reply:
Now handled by `BaseControlledLanguageAdminOnlySiteContentAPITest` as in the comment above